docs/bsp.md

```diff
@@ -341,31 +348,31 @@ Notification:
 
 ```scala
 trait ShowMessageParams {
-	/** The message type. See {@link MessageType}. */
-	def type: Int
+  /** The message type. See {@link MessageType}. */
+  def `type`: Int
```

Comment:
Let's keep `type` un-backticked to avoid confusion for non-Scala readers that the backticks are not expected in JSON object keys.